components/search/src/filter.rs

```diff
@@ -183,8 +183,26 @@ mod tests {
                         }]),
                         search_term_param_name: Some("baz".to_string()),
                     },
-                    suggestions: None,
-                    trending: None,
+                    suggestions: Some(JSONEngineUrl {
+                        base: "https://example.com/suggestions".to_string(),
+                        method: Some(crate::JSONEngineMethod::Get),
+                        params: Some(vec![SearchUrlParam {
+                            name: "lang".to_string(),
+                            value: None,
+                            experiment_config: Some("lang-param".to_string()),
+                        }]),
+                        search_term_param_name: Some("suggest".to_string()),
+                    }),
+                    trending: Some(JSONEngineUrl {
+                        base: "https://example.com/trending".to_string(),
+                        method: Some(crate::JSONEngineMethod::Get),
+                        params: Some(vec![SearchUrlParam {
+                            name: "trend".to_string(),
+                            value: Some("now".to_string()),
```

Comment:
How about changing `trend` to `trend-name`and `now` to `trend-value` might be more clear?